src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -14,7 +14,7 @@ public static partial class Http
 
             public static string SecureHost => GetValue("COREFX_SECUREHTTPHOST", DefaultAzureServer);
 
-            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "http2.akamai.com");
+            public static string Http2Host => GetValue("COREFX_HTTP2HOST", "www.microsoft.com");
```

Comment:
I'd think it would be a difference we want to test against.  If there's currently a bug in the client such that one server works and another doesn't, it would seem there's a meaningful difference in the behaviors of the servers, both of which are acceptable and should work, and thus we should ideally have tests that verify our clients handle them both correctly.